tensor2tensor/rl/envs/tf_atari_wrappers.py

```diff
@@ -25,6 +25,7 @@
 from tensor2tensor.layers import discretization
 from tensor2tensor.models.research import autoencoders
 from tensor2tensor.rl.envs.in_graph_batch_env import InGraphBatchEnv
+from tensor2tensor.utils import trainer_lib
```

Comment:
This is a huge import to otherwise mostly self-standing file. Let's do the other changes in one PR first and then consider this.

Reply:
Of course. This import is needed just for the `add_problem_hparams` function, which sets `problem` and `problem_hparams` in model `hparams`. I used this function to avoid code repetition, but I can instead manually set those things to avoid this dependency.